packages/components/src/sandbox/index.native.js

```diff
@@ -0,0 +1,249 @@
+/**
+ * External dependencies
+ */
+import { Dimensions, Platform } from 'react-native';
+import { WebView } from 'react-native-webview';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	renderToString,
+	useRef,
+	useState,
+	useEffect,
+} from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import sandboxStyles from './style.scss';
+
+const observeAndResizeJS = `
+	( function() {
+		var observer;
+
+		if ( ! window.MutationObserver || ! document.body || ! window.parent ) {
+			return;
+		}
+
+		function sendResize() {
+			var clientBoundingRect = document.body.getBoundingClientRect();
+
+            window.ReactNativeWebView.postMessage(JSON.stringify( {
```

Comment:
The [function `postMessage`](https://github.com/react-native-webview/react-native-webview/blob/master/docs/Reference.md#postmessagestr) is exposed by the `react-native-webview` library to [communicate between RN and the WebView](https://github.com/react-native-webview/react-native-webview/blob/master/docs/Guide.md#communicating-between-js-and-native), in this case, we use for notifying resize changes.

Reply:
Should we maybe add this as a comment here in code?

Reply:
Sure, I'll add it 👍 .

Reply:
In [this commit](https://github.com/WordPress/gutenberg/pull/33426/commits/e5251cb2ad2ba8cfa95e4fc3eb91522fcda94e8d) I added the mentioned comment.